rpc_util.go

```diff
@@ -181,6 +183,24 @@ func FailFast(failFast bool) CallOption {
 	})
 }
 
+// CallCodec returns a CallOption that uses the given Codec for the RPC call.
+// This will override the Codec assocated with the ClientConn.
+func CallCodec(codec Codec) CallOption {
+	return beforeCall(func(c *callInfo) error {
+		c.codec = codec
+		return nil
+	})
+}
+
+// CallContentSubtype returns a CallOption that will set the content subtype
+// to the given string. Normally, the value returned from Codec.String() is used.
+func CallContentSubtype(contentSubtype string) CallOption {
```

Comment:
In terms of explicitly specifying the codec - don't understand either, other than breaking existing clients that don't properly implement the grpc wire protocol (which I mentioned as a concern). Right now, `proto` is the subtype specified even though users may not realize it (since grpc-go at least makes `protoCodec` the default codec). This is just specifying that over the wire.